x-pack/plugins/endpoint/server/routes/alerts/list/schemas.ts

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { decode } from 'rison-node';
+import { i18n } from '@kbn/i18n';
+import { schema } from '@kbn/config-schema';
+import { esKuery } from '../../../../../../../src/plugins/data/server';
+import { EndpointAppConstants } from '../../../../common/types';
+
+export const alertListReqSchema = schema.object(
```

Comment:
A lot of these fields would apply to other collections that use this paging API. It'd be nice to have a parent schema we can mixin with other schemas.